### ISSUE TITLE
date: illegal option on FreeBSD

### ISSUE THREAD

**Issue by albanmaire** [ChaoticWeg/discord.sh#39]
Hi,

I use `discord.sh` on FreeBSD, everything if fine except when `--timestamp` is called.

The FreeBSD `date` program does not understand the `--iso-8601` argument and returns this to the console:
```shell
# date -u --iso-8601=seconds
date: illegal option -- -
usage: date [-jnRu] [-d dst] [-r seconds|file] [-t west] [-v[+|-]val[ymwdHMS]]
            [-I[date | hours | minutes | seconds]]
            [-f fmt date | [[[[[cc]yy]mm]dd]HH]MM[.ss]] [+format]
```

### Quick fix for FreeBSD's users
Get the line:
```shell
get_ts() { date -u --iso-8601=seconds; };
```

Replace with this:
```shell
get_ts() { date -u -Iseconds; };
```

Best regards,
al.

**Comment by fieu** [comment]
Hey @albanmaire,

I tested your correction on a couple different Linux systems and it seems to provide the same result without error. Turns out your correction also fixes the same error which happens on macOS.

I've gone ahead and updated the script with your correction (2f2c0f4c14b48474a6a360eb401c040016cfbedb). 

There's a new minor release containing the fix (https://github.com/ChaoticWeg/discord.sh/releases/tag/v1.6.1).

Thanks for the catch!
